src/patternfly/demos/PasswordGenerator/examples/PasswordGenerator.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 'Password generator'
+beta: false
+section: demos
+---
+
+## Examples
+
+### Provide a generated password
+
+```hbs
+  {{#> form form--id="password-generator-demo--initial"}}
+    {{#> form-group}}
+      {{#> form-group-label form-group-label-info="true"}}
+        {{#> form-group-label-main}}
+              {{#> form-label form-label--attribute=(concat 'for="' form--id '-password"') required="true"}}Password{{/form-label}}
+          {{> form-group-label-help}}
+        {{/form-group-label-main}}
+        {{#> form-group-label-info}}
+        {{/form-group-label-info}}
+      {{/form-group-label}}
+      {{#> form-group-control}}
+        {{#> input-group}}
+            {{#> form-control controlType="input" input="true"  form-control--attribute=(concat 'required type="text" id="' form--id '-password" name="' form--id '-password" aria-label="Password input" value="" placeholder="Password"')}}
+          {{/form-control}}
+          {{#> button button--modifier="pf-m-control" button--attribute='aria-label="Show password button for password input"'}}
```

Comment:
Should we set the password to hidden by default? I think that's how I would expect it. Also I think the design has the icon wrong. If the password is hidden w/ `type="password"`, the icon should be the regular eye. And if it's visible w/ `type="text"`the icon should be the eye with the line through it, as you have it here.

Also this should probably just say "Show password" as the action the button performs. Also looks like we should update the password strength demos. This will match the login page demo with the show/hide password button - https://www.patternfly.org/v4/components/login-page/react/showhide-password/